src/Uno.Extensions.Authentication.UI/Web/WebAuthenticationProvider.cs

```diff
@@ -45,6 +45,12 @@ ITokenCache Tokens
 			authData = authData.Substring(idx + 1);
 		}
 
+		idx = authData.IndexOf("#");
+		if (idx >= 0 && idx < authData.Length - 1)
+		{
+			authData = authData.Substring(idx + 1);
+		}
+
```

Comment:
Would it be possible to use `Uri` parts to parse that string ?

Reply:
done